src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -5,26 +5,19 @@ sidebar_order: 3
 
 {% include learn-sdk.md platform="javascript" %}
```

Comment:
Out of scope for this PR, but can we give the old deprecated SDK a less current looking URL than https://docs.sentry.io/clients/javascript/ ?  Why is one a "platform" and one a "client"?